assets/src/index.js

```diff
@@ -3,12 +3,17 @@ import ReactDOM from 'react-dom'
 import { BrowserRouter as Router, Route } from 'react-router-dom'
 import './index.css'
 import App from './containers/App'
+import CourseList from './containers/CourseList'
 import 'typeface-roboto'
 // import * as serviceWorker from './serviceWorker'
 
 ReactDOM.render(
-  <Router basename='/courses/'>
-    <Route path='/:courseId' component={App} />
+  <Router basename='/'>
```

Comment:
This commit seems to resolves the warning message
`tiny-warning.esm.js:13 Warning: You are attempting to use a basename on a page whose URL path does not begin with the basename. Expected path "/" to begin with "/courses".`